.github/workflows/ci.yml

```diff
@@ -135,3 +138,63 @@ jobs:
         env:
           OPERATOR_IMAGE_TAG: ${{ env.SHORT_SHA }}
         run: make buildx-image
+
+  build-fips:
+    if: ${{ (github.event_name == 'pull_request' || github.ref_name  == 'master') && github.actor != 'dependabot[bot]' && env.ENABLE_FIPS_BUILDS == 'true' }}
```

Comment:
The `env` context is not available under `jobs.<job_id>.if` ([ref](https://docs.github.com/en/actions/reference/contexts-reference#context-availability)) which is why the CI workflow is failing for the PR here: https://github.com/mattermost/mattermost-operator/actions/runs/16366397098

We'll probably need to move the ` env.ENABLE_FIPS_BUILDS == 'true' ` if statement whithin the steps, where the `env` context is available.